tests/integration/test_integration.py

```diff
@@ -46,12 +39,8 @@ async def deploy(ops_test: OpsTest, request):
         application_name=APP_NAME,
         trust=True,
     )
-
-    await _deploy_mongodb(ops_test)
-    await _deploy_tls_provider(ops_test)
     await _deploy_nms_mock(ops_test)
```

Comment:
or we can have only 1 mock charm what implements fiveg_core_gnb and fiveg_n2 integrations

Reply:
I prefer to keep them separate. It will reflect the real scenario better.